README.md

```diff
@@ -1,13 +1,59 @@
-# Evictions Moratoriums Map
+# Emergency Tenants Protections Map
 
-Mapping states, counties, and municipalities that are enacting a moratorium on evictions due to the corona virus pandemic.
+Mapping states, counties, and municipalities that are enacting emergency tenant protections due to the corona virus pandemic.
 
 ## Data Source
 
 Data sourcing is being provided by the [Anti-Eviction Mapping Project](https://www.antievictionmap.com/).
 
-## Running locally
+## Developer Instructions
 
-Run a simple server, e.g. `python -m SimpleHTTPServer` if you have Python.
+Requires [NodeJS](https://nodejs.org/en/) >= v10.13 and [NPM](https://www.npmjs.com/) >= 6.4 installed and available on the [CLI](https://en.wikipedia.org/wiki/Command-line_interface). (Older versions of Node and NPM may or may not work. To manage multiple NodeJS and NPM versions you may use [NVM](https://github.com/nvm-sh/nvm)).
```

Comment:
I should mention Yarn here, as I've been using that for install dependencies instead of npm.